src/pages/Categories.js

```diff
@@ -0,0 +1,11 @@
+function Catagories() {
+  return (
+    <>
+      <div className="h-4/5 flex items-center justify-center">
+        <button type="button" className="bg-sky-400 text-white rounded py-2 px-16 mt-2px">Update Progress</button>
```

Comment:
- Optional
Since you are meeting all the requirements, and the project looks good, I am going to mark this as optional even though it is a requirement of the project. However, Kindly change the text `Update Progress` to `Check status. 
![image](https://user-images.githubusercontent.com/33268927/190863765-a4c18bd1-c71e-4195-9552-a9518eb15623.png)